spanner_orm/condition.py

```diff
@@ -93,6 +96,13 @@ def __init__(self, origin_column, destination_model, destination_column):
     self.destination_model = destination_model
     self.destination_column = destination_column
 
+  def __eq__(self, obj):
+    return (super().__eq__(obj) and
```

Comment:
hmm, not a huge fan of calling parent __eq__, but i'm not sure of a cleaner way to do all this. these new methods are not very clean right now...